packages/core/config/irvingConfigSchema.js

```diff
@@ -22,4 +22,5 @@ module.exports = {
   stylelintConfig: [() => {}],
   styleguideConfig: [() => {}],
   styleguideSetup: [],
+  trailingSlashBlacklist: [],
```

Comment:
See https://alleyinteractive.slack.com/archives/C8LFQ1S13/p1580421913350800 about the naming here